src/views/Main/Main.test.js

```diff
@@ -0,0 +1,39 @@
+import {
+  screen,
+  render,
+  waitForElementToBeRemoved,
+} from '@testing-library/react';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import App from '../../App';
+
+const character = {
+  results: [
+    {
+      id: 1,
+      name: 'Rick Sanchez',
+      status: 'Alive',
+      gender: 'Male',
+      species: 'Human',
+    },
+  ],
+};
+
+const server = setupServer(
+  rest.get('https://rickandmortyapi.com/api/character', (req, res, ctx) =>
+    res(ctx.json(character))
+  )
+);
+
+beforeAll(() => server.listen());
+afterAll(() => server.close());
+
+test('should render Rick Sanchez', async () => {
+  render(<App />);
+  screen.getByText(/loading.../i);
+  screen.debug();
+
+  const charName = await screen.findByText('Rick Sanchez');
+  expect(charName).toBeInTheDocument();
+  screen.debug();
```

Comment:
In our behavioral test, we will also want to test for the search bar and then have a userEvent that types into the search bar and renders our mock data. It is also a good idea to test for specific mock data after the search is completed.